site/content/blog/2021-09-01-whats-new-in-svelte-september-2021.md

```diff
@@ -0,0 +1,80 @@
+---
+title: What's new in Svelte: September 2021
+description: StackOverflow's most loved web framework
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+This month, Svelte was [voted StackOverflow's most loved web framework](https://insights.stackoverflow.com/survey/2021#section-most-loved-dreaded-and-wanted-web-frameworks), Tan Li Hau [talked to Svelte Radio](https://share.transistor.fm/s/84c7521b) about his [Svelte-filled YouTube channel](https://www.youtube.com/channel/UCbmC3HP3FaAFdcZkui8YoMQ), and SvelteKit got closer than ever to 1.0!
+
+## New in Svelte
+- `use:actions` can now be used on `<svelte:body>` (**3.42.0**)
+- `HTMLElement`, `SVGElement` (**3.42.2**) and `BigInt` (**3.42.3**) are now known globals
+- Less code in Svelte's output thanks to the following improvements in **3.42.2**:
+  - Collapse whitespace in class and style attributes
+  - Deselect all <option>s in a <select> where the bound value doesn't match any of them
+  - In hydrated components, only rely on helpers for creating the types of elements present in the component
+- Scaling is now accounted for in `flip` animations (**3.42.2**)
+
+
+For a full list of features and bug fixes, check out the [Svelte changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## SvelteKit Updates
+
+Svelte maintainers are [looking for help getting SvelteKit to 1.0](https://github.com/sveltejs/kit/issues/2100). We've knocked out over 100 issues that were on the 1.0 milestone. There's only a couple dozen left and we'd love a hand making that list a bit shorter!
+
+If you'd like to help, please consider working on any of the [1.0 milestone issues](https://github.com/sveltejs/kit/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0).
+
+Some noteable changes in SvelteKit in the last month include...
+
+- SvelteKit will now detect if a prerendered app is trying to access a query paremter and return an error - instead of silently failing ([#2104](https://github.com/sveltejs/kit/pull/2104))
```

Comment:
```suggestion
- SvelteKit will now detect if a prerendered app is trying to access a query parameter and return an error - instead of silently failing ([#2104](https://github.com/sveltejs/kit/pull/2104))
```

Reply:
This was marked resolved but doesn't seem to be fixed.